evap/settings_test.py

```diff
@@ -5,22 +5,22 @@
         "NAME": "evap",
         "USER": "postgres",
         "PASSWORD": "postgres",
-        "HOST": "postgres",
+        "HOST": "localhost",
```

Comment:
Was this necessary? I think the dedicated per-service hostname has the advantage that in theory, the workflow could use a "service" hosted in a different container. We don't currently use that, but I wouldn't pessimize without a reason :D

Reply:
Yes it is. When no container: is specified, the steps are runned directly on the runner, but postgres and redis still inside a docker container, with the defined port mapping. So, the jobs needs to reference localhost to access the docker containers.
(see [here](https://docs.github.com/en/actions/using-containerized-services/creating-postgresql-service-containers#running-jobs-directly-on-the-runner-machine))